staging/src/k8s.io/kubectl/pkg/cmd/debug/debug.go

```diff
@@ -260,7 +279,8 @@ func (o *DebugOptions) Run(f cmdutil.Factory, cmd *cobra.Command) error {
 			opts.Config = config
 			opts.AttachFunc = attach.DefaultAttachFunc
 
-			if err := handleAttachPod(ctx, f, o.podClient, debugPod.Namespace, debugPod.Name, containerName, opts); err != nil {
```

Comment:
I think this may be an accidental revert during rebase.

Reply:
good catch